draft-ietf-ppm-dap.md

```diff
@@ -995,7 +1002,6 @@ standard tokens for use in the "type" field:
 | invalidBatchSize            | There are an invalid number of reports in the batch. |
 | invalidAggregationParameter | The aggregation parameter assigned to a batch is invalid. |
 | batchMismatch               | Aggregators disagree on the report shares that were aggregated in a batch. |
-| unauthorizedRequest         | Authentication of an HTTP request failed (see {{request-authentication}}). |
```

Comment:
This error type was never used, and besides, doing this at the DAP layer is inappropriate:

- in some deployments, authentication may happen before a request ever gets to a DAP implementation (e.g. in a load balancer or other proxy) so mandating a DAP specific problem document is a burden
- if you're using something like mTLS for auth, auth failures won't be represented as HTTP responses at all, much less a DAP specific problem document

Reply:
IIRC this is a relic of when we were doing HMAC in band.